### ISSUE TITLE
Superset not loading module cc_superset

### ISSUE THREAD

**Issue by rad-pat** [datafuselabs/databend-sqlalchemy#29]
```ERROR:superset.db_engine_specs:Unable to load Superset DB engine spec: databend
Traceback (most recent call last):
  File "/app/superset/db_engine_specs/__init__.py", line 78, in load_engine_specs
    engine_spec = ep.load()
  File "/usr/local/lib/python3.10/site-packages/importlib_metadata/__init__.py", line 184, in load
    module = import_module(match.group('module'))
  File "/usr/local/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 992, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1004, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'databend_sqlalchemy.cc_superset'

**Comment by hantmac** [comment]
Which superset version are you using?  I build the superset docker image according to :

```
FROM apache/superset
# Switching to root to install the required packages
USER root
RUN pip install databend-py
RUN pip install databend-sqlalchemy==0.3.2
# Switching back to using the `superset` user
USER superset
```

And visit http://localhost:8080 is ok.

**Comment by hantmac** [comment]
@rad-pat  I got you, since from 0.3.2, the databend-sqlalchemy use `databend-driver`(rust binding) as internal driver, so the `sslmode` param was not handled in superset, which is `secure` insteam. Maybe should we make pr to superset to change the `secure` to `sslmod`?

**Comment by rad-pat** [comment]
@hantmac, yes. I can make that PR if you like and tag you as reviewer? We're currently trying to get everything working with superset.

**Comment by hantmac** [comment]
> @hantmac, yes. I can make that PR if you like and tag you as reviewer? We're currently trying to get everything working with superset.

Of course.  Before that PR if you want to use superset for databend, you can use v0.3.2.

**Comment by rad-pat** [comment]
Fixed by #35
